src/rustdoc_schema.graphql

```diff
@@ -695,6 +695,61 @@ type Constant implements Item & Importable & GlobalValue {
   attrs: [String!]!
   visibility_limit: String!
 
+  # properties for Constant
+  """
+  The expression of the constant, if any, as a Rust literal or `"_"`. For example:
+  ```rust
+  //                                 // expr
+  const MIN     : usize = 16       ; // 16
+  const MIN_SIZE: usize = MIN      ; // "MIN", referring to the other constant's name
+  const LOG_AS  : &str  = "batch"  ; // "\"batch\"", including escaped quotes
+  const YEAR    : Years = Years(42); // "_"
+  const EXPR_2_2: i32   = 2 + 2    ; // "_"
+  const FN_FIVE : i32   = five()   ; // "_"
+  const fn five() -> i32 { 5 };
+  struct Years(i32);
+  ```
+  If the constant is set:
+
```

Comment:
minor formatting nit:
```suggestion

  If the constant is set:
```

Reply:
This is actually intentional since no space after the "header" breaks the formatting, so instead of 2 bullet points you get one line (at least in my Sublime editor with LSP plugin)

Not sure how it works in your wasm playground or any other tools

Let me know if you want to add an extra empty line before anyway (personally prefer a bit more compact, but up to you)

Reply:
Ah nice, thanks for double-checking it in the editor.

Let's add the extra empty line before the block, though. I think that way it's a bit more readable when it's not rendered as markdown.

Reply:
It's just the adding the newline, so I'll merge this as-is and add the newline in a separate PR and then cut a release so we can move faster.